src/ut/realmmanager_test.cpp

```diff
@@ -118,7 +138,10 @@ TEST_F(RealmmanagerTest, CreateDestroy)
                                                  DIAMETER_REALM,
                                                  DIAMETER_HOSTNAME,
                                                  2,
-                                                 _mock_resolver);
+                                                 _mock_resolver,
+                                                 NULL,
+                                                 NULL,
```

Comment:
It looks like you've changed this so the PDLogs are passed by ref not pointer. You'll need to fix this up here.